modules/sno-du-removing-the-console-operator.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-removing-the-console-operator_{context}"]
+= Disabling the console Operator
+
+The console-operator installs and maintains the web console on a cluster. When the node is centrally managed the Operator is not needed and makes space for application workloads.
+
+.Procedure
+
+* You can disable the Operator using the following configuration file.
+No configuration changes are needed. Use the provided settings:
++
+[source,yaml]
+----
```

Comment:
I'm not sure about this one. On my cluster, the config is different.

Reply:
@imiller0 - Please verify.

Reply:
Looks correct.